logconv.m

```diff
@@ -33,7 +33,7 @@ function logconv()
 % 2: import the data from the .mat file
 %    (requires the generated .mat file)
 % else: Defaults to 0
-loadingMode = 0;
+loadingMode = 2;
```

Comment:
should be changed back to the default value

Reply:
Already changed back